vsts-extension-google-play.json

```diff
@@ -2,7 +2,7 @@
     "manifestVersion": 1.0,
     "id": "google-play",
     "name": "Google Play",
-    "version": "4.201.0",
+    "version": "4.201.1",
```

Comment:
Could you please bump minor task version to the [current sprint](https://whatsprintis.it/) - 204?